VK/VK/ShowGriendPhotoScreen/ShowPhotoView.swift

```diff
@@ -0,0 +1,12 @@
+// ShowPhotoView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Представление - слайд для скролла фотка.
+final class ShowPhotoView: UIView {
+    // MARK: - Public @IBOutlet
+
+    @IBOutlet var photoImageView: UIImageView!
+    @IBOutlet var descriptionLabel: UILabel!
```

Comment:
```suggestion
    @IBOutlet private var photoImageView: UIImageView!
    @IBOutlet private var descriptionLabel: UILabel!
```
до них можно достучаться через экземпляр класса и они будут приватными

Reply:
Ругается. Пока оставлю так, буду разбираться